contrib/julia-mode.el

```diff
@@ -356,8 +356,9 @@ This variable has a moderate effect on indent performance if set too
 high.")
 
 (defun julia-paren-indent ()
-  "Return the column position of the innermost containing paren
-before point. Returns nil if we're not within nested parens."
+  "Return the column position of the first non-blank character
+after the innermost containing paren before point. Returns nil if
```

Comment:
You're absolutely right that the previous docstring had an off-by-one error, however I find this longer sentence hard to mentally parse. How about "Return the column position required to line up with the containing paren."?